src/lang/cpp/internal/CDTParser.java

```diff
@@ -126,12 +122,8 @@ public class CDTParser {
 	public CDTParser(IValueFactory vf, IRascalValueFactory rvf, PrintWriter stdOut, PrintWriter stdErr, TypeStore ts,
 			IRascalMonitor monitor, IList stdLib, IList includePath, IMap additionalMacros, boolean includeStdLib) {
 		this.vf = vf;
```

Comment:
Maybe remove the corresponding constructor arguments as well?

Reply:
done